src/views/router_views/company/confirm/confirm.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+
+{% block main_content %}
+
+{% set address %}
+    {{companyProfile.registeredOfficeAddress.addressLineOne}}
+    <br/>
+    {{companyProfile.registeredOfficeAddress.addressLineTwo}}
+    <br/>
+    {{companyProfile.registeredOfficeAddress.postalCode}}
+{% endset %}
+
+<div class="govuk-grid-row">
+
+
+   <h1 class="govuk-heading-xl">Confirm this is the correct company</h1>
+
+            <h1 class="govuk-heading-l">{{companyProfile.companyName}}</h1>
+
+
+ 
+
+{{ govukSummaryList({
+  rows: [
+    {
+      key: {
+        text: "Company number"
+      },
+    value: {
+            text: companyProfile.companyNumber
+      }
+    },
+    {
+      key: {
+        text: "Status"
+      },
+      value: {
+        html: companyProfile.companyStatus | capitalize
+      }
+    },
+        {
+      key: {
+        text: "Incorporation date"
+      },
+      value: {
+        text: companyProfile.dateOfCreation | formatDate
+      }
+    },
+        {
+      key: {
+        text: "Company type"
+      },
+      value: {
+        text: companyProfile.type | formatProfileType | capitalize
+      }
+    },
+        {
+      key: {
+        text: "Registered office address"
+      },
+      value: {
+        html: address
+      }
+    },
+        {
+      key: {
+        text: "Last accounts made up to"
+      },
+      value: {
+      text: companyProfile.accounts.nextAccounts.periodStartOn | formatDate
+      }
+    },
+        {
+      key: {
+        text: "Next accounts due"
+      },
+      value: {
+      html: companyProfile.accounts.nextDue | formatDate
+      }
+    }
+
+  ]
+}) }}
+
+
+   {{ govukButton({
+  text: "Confirm and continue",
+  href: uploadLink
+}) }}  
+
+<p><a href="{{ changeCompanyUrl }}">Choose a different company</a></p>
```

Comment:
This styling is slightly wrong. I think it's because we are using a slightly older version of gds  than the prototype (4.6.0 vs 4.7.00).
It just needs the following classes to be added to make it look right. 
- `.govuk-body` to the <p> tag
- `govuk-link` to the <a> tag
```suggestion
<p class="govuk-body"><a class="govuk-link" href="{{ changeCompanyUrl }}">Choose a different company</a></p>
```